dataset/src/main/java/org/keycloak/benchmark/dataset/config/DatasetConfig.java

```diff
@@ -119,6 +119,16 @@ public class DatasetConfig {
     @QueryParamIntFill(paramName = "groups-per-transaction", defaultValue = 100, operations = { CREATE_REALMS })
     private Integer groupsPerTransaction;
 
+    @QueryParamFill(paramName = "groups-with-hierarchy", defaultValue = "false", operations = { CREATE_REALMS })
```

Comment:
Is it possible to add some small comments to the 3 new parameters of this PR? See comments for other parameters in this class.

IMO something like you described in the description of this PR is exactly what could be nice here, so the usage is clear without a need to look into the code.

Reply:
Thank you for the review. Added code comments for the new parameters and added the "Create many groups" section in the usage documentation. Let me know it is too wordy.